### DISCUSSION TITLE
Longevity question — If Keka goes down, how do I unencrypt my files?

### DISCUSSION THREAD

**Discussion by user7179** [aonez/Keka#1152] [Q&A] [answered]
Hi there, I was wondering how I would unencrypt my **Keka AES-256 zipped files** if Keka ever goes down permanently.

**Comment by gingerbeardman** [comment, marked as answer]
macOS Archive Utility will expand them no problem, at least it did just now in Monterey.

Also: BetterZip, Entropy, MacZip, The Unarchiver (there are just the apps I have installed)

Not: WinZip for Mac.

<img width="420" alt="Screen shot 2022-08-07 at 08 48 35" src="https://user-images.githubusercontent.com/49612/183280906-efec8445-dae3-46c9-8914-7e2e61f1e3be.png">

**Reply by aonez** [reply]
@user7179 AES-256 is an open standard so there will always be options, plus plus Keka will never go down 😛

**Comment by TechnoMemory** [comment]
You can decrypt with p7zip (similar to 7z but for Unix). 

although apple has built in ways to encrypt and decrypt files and folders. check out these sources for more info if want:
[https://www.youtube.com/watch?v=FY7zARFrpDU](url)
[https://www.intego.com/mac-security-blog/how-to-use-apples-built-in-features-to-encrypt-files-and-folders/](url)
